list.js

```diff
@@ -2,7 +2,7 @@
 /* global lunr */
```

Comment:
Because we're now using a local version of the `$` variable, looks like we can remove it from eslint's globals. -- https://travis-ci.org/ccnmtl/calendar-widget/builds/206782491#L572